### DISCUSSION TITLE
how to use?

### DISCUSSION THREAD

**Discussion by JeetChaurasia** [friction2d/friction#379] [General]
Guys how to use it is there any manuals???i'm new just found it and i'm going crazzyyyyyy it's so goodddddddd

**Comment by AlmostMikhail** [comment]
Try Alexander Kiryanov on YouTube. He does enve but it basically applies to friction but friction is being continuously updated by the community and enve abandoned.

https://youtu.be/VWTjkjfw_Dc?si=KekxtEjQybeJ1l2R

**Reply by JeetChaurasia** [reply]
okay let me check
